tests/model_hub_tests/tensorflow/test_tf_convert_model.py

```diff
@@ -24,6 +26,51 @@ def is_hf_link(link: str):
     return link.startswith("hf_")
 
 
+def get_output_signature_from_keras_layer(model):
+    try:
+        from openvino.frontend.tensorflow.utils import trace_tf_model_if_needed
+        traced_model = trace_tf_model_if_needed(model, None, None, None)
+        return traced_model.structured_outputs
+    except:
+        return None
```

Comment:
please put this routine under the routine responsible for repacking of OV results mentioned below.